components/net/http_loader.rs

```diff
@@ -375,7 +375,7 @@ fn obtain_response(client: &Client<Connector, WrappedBody>,
     let connect_start = precise_time_ms();
     let request = HyperRequest::builder()
         .method(method)
-        .uri(url.clone().into_url().as_ref().replace("|", "%7C"))
+        .uri(url.clone().into_url().as_ref().replace("|", "%7C").replace("{", "%7B").replace("}", "%7C"))
```

Comment:
@seanmonstar What should we do here ?

Reply:
I don't have enough context to answer. What is this trying to do? Percent-encode these characters, it looks like? What's the issue? (It looks like this is encoding both `|` and `}` to `%7C`...)

Reply:
Yeah, I fixed that already, `}` is `%7D`.

Without those replace, hyper doesn't like requests with either of those characters in the query params (meaning it errors and never send the request because the URI fails to parse).

Reply:
So then, it sounds like this make it work, right?

Reply:
Yeah but this feels really hacky...

Reply:
I've filed https://github.com/hyperium/http/pull/247